### PR TITLE
CSP Adapter Bump to version 1.0.0-rc1

### PR DESCRIPTION
**Pull request by MbolotSuse**

### CODE REVIEW

**Comment by maxsokolovsky** [comment]
Do you plan to update the minimum required version in the other PR https://github.com/rancher/rancher/pull/38020?

**Comment by MbolotSuse** [comment]
@maxsokolovsky depends on the order that this gets in. If this goes first I can do that.

**Review by maxsokolovsky — APPROVED** [review]

File: charts/rancher-csp-adapter/1.0.0+up1.0.0-rc1/Chart.yaml [hunk 1]
```diff
@@ -10,8 +10,8 @@ annotations:
   catalog.cattle.io/release-name: rancher-csp-adapter
   catalog.cattle.io/scope: management
 apiVersion: v2
-appVersion: 0.1.0-rc8
+appVersion: 1.0.0-rc1
 description: Cloud Service Provider Marketplace Adapter for Rancher. Monitors Rancher
   entitlements against usage.
 name: rancher-csp-adapter
-version: 0.1.0+up0.1.0-rc8
+version: 1.0.0+up1.0.0-rc1
```
Comment (cbron) [inline charts/rancher-csp-adapter/1.0.0+up1.0.0-rc1/Chart.yaml:17]: We were right that there is not a `kube-version` in here, only the annotation

**Review by cbron — APPROVED** [review]

**Comment by MbolotSuse** [comment]
Related to rancher/rancher#37918

**Pull request merged**